Tasks/PublishTestResultsV2/make.json

```diff
@@ -2,7 +2,7 @@
   "externals": {
     "archivePackages": [
       {
-        "url": "https://testmanagementstore.blob.core.windows.net/testmanagementptrcontainer/29161277/PublishTestResults.zip",
+        "url": "https://testmanagementstore.z13.web.core.windows.net/testmanagementptrcontainer/29161277/PublishTestResults.zip",
```

Comment:
What is this URL?

Reply:
We are using azure web static container from same storage account.

[Static website hosting in Azure Storage | Microsoft Learn](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-blob-static-website)